lib/service/cfg.go

```diff
@@ -706,6 +706,8 @@ type DatabaseAWS struct {
 	RDS DatabaseAWSRDS
 	// ElastiCache contains ElastiCache specific settings.
 	ElastiCache DatabaseAWSElastiCache
+	// SecretStore contains settings for managing secrets.
+	SecretStore types.SecretStore
```

Comment:
yes. can certainly create a type here. I did add a separate type for file config. Was just wondering if it will bring much benefit to the service config.